Makefile

```diff
@@ -30,10 +30,12 @@ help:
 	@echo " * 'lint' - Execute the source code linter"
 	@echo " * 'gofmt' - Verify the source code gofmt"
 
-lint:
+ensure-gopath:
```

Comment:
Suggest the name 'check-gopath' instead.

Also, we could put the .PHONY tags above each target, to make it easier to keep 'em in sync?